eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -0,0 +1,43 @@
+param(
+  $RepoOwner,
+  $RepoName,
+  $BranchPrefix,
+  $WorkingDirectory,
+  $AuthToken
+)
+
+. "${PSScriptRoot}\common.ps1"
+
+pushd $WorkingDirectory
+git clone https://github.com/$RepoOwner/$RepoName.git
+pushd $RepoName
+$syncBranches = git branch -r --list origin/$BranchPrefix* | % { $_ -replace "origin/", "" }
```

Comment:
Can you just use this API to query for branches and delete them without having to clone the repo?

https://docs.github.com/en/free-pro-team@latest/rest/reference/git#delete-a-reference

Should be much faster.

Reply:
Thanks, I looked everywhere for this. Weird I could not find it.